### ISSUE TITLE
Large Cluster jewel not working

### ISSUE THREAD

**Issue by MeavesMev** [PathOfBuildingCommunity/PathOfBuilding#630]
UPDATE: When I craft the jewel in PoB it works just fine, just importing it doesn't work at all so I have to recraft it everytime or just not delete old items.

My large cluster jewel isn't working and expanding the tree. Medium and small jewels work just fine.
You can see the jewel expanding in the small window.
Large Jewel: 
![LargeCluster](https://user-images.githubusercontent.com/62435840/77176858-f7198300-6ac4-11ea-9183-e77749d1ce78.png)

Medium Jewel:
![MediumCluster](https://user-images.githubusercontent.com/62435840/77176945-16b0ab80-6ac5-11ea-92f7-be1bb1067c30.png)

And on the tree it looks like this:
![LargeCluster2](https://user-images.githubusercontent.com/62435840/77177341-a22a3c80-6ac5-11ea-81a0-64cec52d7e10.png)

![MediumCluster_SmallCluster](https://user-images.githubusercontent.com/62435840/77177392-b53d0c80-6ac5-11ea-9e77-c91de98096da.png)

**Comment by ppoelzl** [comment]
Could you please provide your build/character that you tested this with?

**Comment by nicooo** [comment]
It seems to be a problem with claw/dagger attack jewels. It happens to me with this one:

```
Rarity: RARE
Hypnotic Vessel
Large Cluster Jewel
Unique ID: c8be88fdb6200c6a2cf657a981994d7d75e5fb06942c5716688fe1d2b56febb3
Item Level: 56
Implicits: 2
{crafted}Adds 8 Passive Skills
{crafted}Added Small Passive Skills grant: Claw Attacks deal 12% increased Damage with Hits and Ailments Added Small Passive Skills grant: Dagger Attacks deal 12% increased Damage with Hits and Ailments
1 Added Passive Skill is Feed the Fury
1 Added Passive Skill is Wind-up
1 Added Passive Skill is a Jewel Socket
```

**Comment by MeavesMev** [comment]
Sorry I didn't check my mails in a few days.  Heres the pastebin: https://pastebin.com/ZJVMcbBh

**Comment by prog-r-amer** [comment]
Happens to me too, super annoying. Also don't know how to transform the jewel into one working without recrafting. Mods look almost identical.

**Comment by reeplan** [comment]
Same thing with Bows enchantments

![image](https://user-images.githubusercontent.com/20599992/79044928-e2866180-7c10-11ea-9859-6cc7949e38c5.png)
```

Damnation Essence
Large Cluster Jewel
Unique ID: fd7368b101250037cacd62ee721e90b3ebefeaefc37307ab9345b9a227371c4b
Item Level: 46
Implicits: 2
{crafted}Adds 11 Passive Skills
{crafted}Added Small Passive Skills grant: 12% increased Damage with Bows Added Small Passive Skills grant: 12% increased Damage Over Time with Bow Skills
1 Added Passive Skill is Drive the Destruction
1 Added Passive Skill is Heavy Hitter
1 Added Passive Skill is Tempered Arrowheads
1 Added Passive Skill is a Jewel Socket
```

**Comment by LocalIdentity** [comment]
Fixed in 1.4.169.1
